src/com/service/PersonService.java

```diff
@@ -34,7 +33,7 @@ private boolean isPersonCreated(Person person) {
 
     @Override
     public boolean addNewPerson(Person person) {
-        if(isPersonCreated(person)) {
+        if (isPersonCreated(person) || person == null) {
```

Comment:
Maybe person== null check should be in a separate if condition. If the user passes null he gets the message that person exists which is not true.

Reply:
I've seen this when testing, but forgot to fix it